tests/data/test_optimization_indexset.py

```diff
@@ -34,157 +34,160 @@ def df_from_list(indexsets: list):
         indexset_1, indexset_2 = create_indexsets_for_run(
-            platform=test_mp, run_id=run.id
+            platform=platform, run_id=run.id
         )
-
-        test_mp.backend.optimization.indexsets.add_elements(
-            indexset_id=indexset_1.id, elements=test_elements
+        platform.backend.optimization.indexsets.add_elements(
+            indexset_id=indexset_1.id,
+            elements=test_elements,  # type: ignore
         )
-        indexset_1 = test_mp.backend.optimization.indexsets.get(
-            run_id=run.id, name=indexset_1.name
+        indexset_1 = platform.backend.optimization.indexsets.get(
+            run_id=run.id, name="Indexset 1"
         )
 
-        test_mp.backend.optimization.indexsets.add_elements(
-            indexset_id=indexset_2.id, elements=test_elements
+        platform.backend.optimization.indexsets.add_elements(
+            indexset_id=indexset_2.id,
+            elements=test_elements,  # type: ignore
         )
 
         assert (
             indexset_1.elements
-            == test_mp.backend.optimization.indexsets.get(
-                run_id=run.id, name=indexset_2.name
+            == platform.backend.optimization.indexsets.get(
+                run_id=run.id, name="Indexset 2"
             ).elements
         )
 
         with pytest.raises(ValueError):
-            test_mp.backend.optimization.indexsets.add_elements(
+            platform.backend.optimization.indexsets.add_elements(
                 indexset_id=indexset_1.id, elements=["baz", "foo"]
             )
 
         with pytest.raises(ValueError):
-            test_mp.backend.optimization.indexsets.add_elements(
+            platform.backend.optimization.indexsets.add_elements(
                 indexset_id=indexset_2.id, elements=["baz", "baz"]
             )
 
-        test_mp.backend.optimization.indexsets.add_elements(
+        platform.backend.optimization.indexsets.add_elements(
             indexset_id=indexset_1.id, elements=1
         )
-        indexset_3 = test_mp.backend.optimization.indexsets.get(
+        indexset_3 = platform.backend.optimization.indexsets.get(
             run_id=run.id, name=indexset_1.name
         )
-        test_mp.backend.optimization.indexsets.add_elements(
+        platform.backend.optimization.indexsets.add_elements(
             indexset_id=indexset_2.id, elements="1"
         )
-        indexset_4 = test_mp.backend.optimization.indexsets.get(
+        indexset_4 = platform.backend.optimization.indexsets.get(
             run_id=run.id, name=indexset_2.name
         )
         assert indexset_3.elements != indexset_4.elements
         assert len(indexset_3.elements) == len(indexset_4.elements)
 
         test_elements_2 = [1, "2", 3.14]
-        indexset_5 = test_mp.backend.optimization.indexsets.create(
-            run_id=run.id, name="Indexset 5"
+        indexset_5 = platform.backend.optimization.indexsets.create(
+            run_id=run.id, name="IndexSet 5"
```

Comment:
```suggestion
            run_id=run.id, name="Indexset 5"
```
If we spell out the name, let's be consistent.
Same below.